system-variables.md

```diff
@@ -427,6 +427,17 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - Default value: ON
 - This variable controls whether to record the execution information of each operator in the slow query log.
 
+### tidb_enable_enhanced_security
```

Comment:
Is this variable readonly?

Reply:
@morgo PTAL

Reply:
Yes it is. Or more specifically it is `Scope: None`, which is the same as read-only.

Reply:
> TiDB system variables behave similar to MySQL with some differences, in that settings might apply on a SESSION, INSTANCE, or GLOBAL scope, or on a scope that combines SESSION, INSTANCE, or GLOBAL.

> * Changes to GLOBAL scoped variables only apply to new connection sessions with TiDB. Currently active connection sessions are not affected. These changes are persisted and valid after restarts.
> * Changes to INSTANCE scoped variables apply to all active or new connection sessions with the current TiDB instance immediately after the changes are made. Other TiDB instances are not affected. These changes are not persisted and become invalid after TiDB restarts.

The document does not mention any about what `Scope: None` means. Cloud you add some explanations?

Reply:
I agree with you. I will change it tomorrow to say Read Only instead.

(There are some minor differences, but they don't matter to users.)

Reply:
This is improved in https://github.com/pingcap/docs/pull/5647/commits/c8758d7b8c130ed8c52aa331e27110c383fed166 , PTAL again thx :-)